code/weapon/beam.h

```diff
@@ -184,7 +184,8 @@ typedef struct beam {
 	int Beam_muzzle_stamp;
 	int firingpoint;
 
-	float		beam_width;
+	float		beam_collide_width;
+	float		beam_light_width;
```

Comment:
I suggest calling this `beam_visible_width` to be unambiguous.  Since I recently worked in the collision code I first thought of light vs. heavy.

Reply:
Well these numbers really only are used by collision or lighting, I think 'visible' might be confusing since whatever it actually looks like is still determined by the individual widths of the sections.

Reply:
`beam_light_radius` would be relatively consistent with how that value is refereed to where it will ultimately be used.

Reply:
Actually scratch that, because you're setting the beam width that lighting is based off of, not the actual light radius. Which actually is a bit not my favorite path forward, but I'll make a separate comment on that.

Reply:
It still refers to the width of the complete beam, right, not the sections?  I think that's fine, because if you're interested in the sub-widths you would use something like `beam_section_width`.

Reply:
Well, see the whole point of this is trying to clear up the ambiguity of "width of the complete beam". With this PR `beam_light_width` is *only* used for sizing the light cast by the beam, nothing else. This is *sort of* what one may mean by "visible beam", but I personally would take that to mean the width of the individual sections, or more broadly, whichever is the widest. That is relevant to light and collision only insofar as it is the last fallback.

Reply:
Okay.  I'll defer to your understanding then.